src/credentials/statusBarItem.ts

```diff
@@ -49,26 +52,40 @@ function handleDevSettings(devSettings: DevSettings, statusBarItem: vscode.Statu
     }
 }
 
-// Resolves when the status bar reaches its final state
-export async function updateCredentialsStatusBarItem(
-    statusBarItem: vscode.StatusBarItem,
-    credentialsId?: string,
-    color?: vscode.ThemeColor
-): Promise<void> {
-    const connectedMsg = localize(
+function updateItem(statusBarItem: vscode.StatusBarItem): void {
```

Comment:
I deleted the old test file as it really didn't do much. Need to extract some `Auth` test util functions before it's worth writing tests for this.